site/public/css/electronic.css

```diff
@@ -175,9 +175,9 @@ button > i.fas:hover {
 .ta-navlink-cont {
     position: relative;
     margin-right: 4px;
-    border: 2px solid var(--default-black);
+    /*border: 2px solid var(--default-black);*/
```

Comment:
There should be an outline. The main problem is that it does not look good on dark mode, but there should still be an outline